CHANGELOG.md

```diff
@@ -2,11 +2,11 @@
 
 FEATURES:
 
-
+* **New Argument:** `rke_cluster.services.etc.backup_config.timeout` - (Optional/Computed) Set timeout in seconds for etcd backup. Just for RKE v1.2.6 and above
```

Comment:
I think this should be `rke_cluster.services.etc.backup_config.timeout` -> `rke_cluster.services.etcd.backup_config.timeout`

Reply:
Thanks for the typo advice. Updated.